runtimes/peregrine/Cargo.toml

```diff
@@ -94,9 +95,9 @@ parachain-info.workspace = true
 
 # Polkadot dependencies
 pallet-xcm.workspace = true
-xcm.workspace = true
 xcm-builder.workspace = true
 xcm-executor.workspace = true
+xcm.workspace = true
```

Comment:
xcm comes before the others alphabetically, right?

Reply:
I think this is alphabetically sorted :thinking: . `-` is 45 and `.` is 46 in the ASCII table. The ordering is ascending